Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -1185,6 +1185,9 @@ extension SourceKitServer {
   }
 
   private func openDocument(_ note: DidOpenTextDocumentNotification, workspace: Workspace) async {
+    if documentManager.latestSnapshot(note.textDocument.uri) != nil {
+      logger.fault("Document '\(note.textDocument.uri.forLogging)' is already open!")
+    }
```

Comment:
Doesn't DocumentManager already log this?

Reply:
Only after https://github.com/apple/sourcekit-lsp/pull/943

Reply:
Oh, yes it does. Removing it again.